src/Backend/Modules/Mailmotor/Actions/Ping.php

```diff
@@ -24,14 +25,38 @@ public function execute(): void
         parent::execute();
 
         // Successful API connection
-        if ($this->get('mailmotor.subscriber')->ping()) {
+        if ($this->ping()) {
             $this->redirect($this->getBackLink(['report' => 'successful-mail-engine-api-connection']));
         }
 
         $this->resetMailEngine();
         $this->redirect($this->getBackLink(['error' => 'wrong-mail-engine-credentials']));
     }
 
+    private function ping(): bool
+    {
+        $gateway = $this->getContainer()->get('mailmotor.factory')->getSubscriberGateway();
+
+        if (!$gateway->ping($this->getContainer()->getParameter('mailmotor.list_id'))) {
+            return false;
+        }
+
+        $settings = $this->getContainer()->get('fork.settings');
+        foreach (Language::getActiveLanguages() as $language) {
+            $languageListId = $settings->get('Mailmotor', 'list_id_' . $language);
+
+            if ($languageListId === null) {
```

Comment:
Same here, why continue if the id is null? Shouldn't the list be created?

Reply:
fixed in 35de0b801